src/library/Qeury.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 Curvegrid Inc.
```

Comment:
Naming of the file: `Qeury.js` => `Query.js`

Reply:
Thanks! I fixed this https://github.com/curvegrid/multibaas-for-google-sheets/pull/14/commits/805f11ccd1ac6740b3e5235b40ed705808c50880 so I guess this will be fine. :)